### PR TITLE
WLS profiles for OKD nightly run 

### PR DESCRIPTION
**Pull request by maggiehe00**
Using new profile for 
sequential run 
srg: https://build.weblogick8s.org:8443/job/wko-okd-test/204/  1 failure
mrg: https://build.weblogick8s.org:8443/job/wko-okd-test/194/  2 failures
parallel run
srg: https://build.weblogick8s.org:8443/job/wko-okd-test/205/    2 failures
fmw: https://build.weblogick8s.org:8443/job/wko-okd-test/201/   1 failure

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -411,7 +474,7 @@
                   **/ItUsabilityOperatorHelmChart
                 </includes-failsafe>
             </properties>
-        </profile>
+        </profile-->
```
Comment (anpanigr) [inline integration-tests/pom.xml:477]: Remove commented code

Comment (maggiehe00) [reply integration-tests/pom.xml]: Done. Thanks!

**Review by vanajamukkara — APPROVED** [review]
LG

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**